services/prometheus/internal/config/global.go

```diff
@@ -0,0 +1,18 @@
+package config
+
+import "github.com/prometheus/common/model"
```

Comment:
The only import left from prometheus because of custom durations format. Can be removed if we copy parsing function from `prometheus/common`, however, we have transitive depenendency on that package anyway. So I think it's ok for now to use this type.